elastic/kibana
# [maps] fix layer error in map embeddable error makes map unusable

Fixes https://github.com/elastic/kibana/issues/155773

https://github.com/elastic/kibana/pull/134243 surfaced layer errors as `output.error`. This was a mistake and caused map embeddable to render only the layer error instead of the map
<img width="500" alt="Screen Shot 2023-04-26 at 7 27 00 AM" src="https://user-images.githubusercontent.com/373691/234602920-4d063a4b-bc02-4fac-9037-0774a790b471.png">

PR resolves issue by no longer surfacing layer errors as `output.error`. The map legend does a great job of surfacing individual layer errors and still keeps the map usable.
<img width="500" alt="Screen Shot 2023-04-26 at 7 26 42 AM" src="https://user-images.githubusercontent.com/373691/234603008-8c24674f-21ca-4046-88ed-b1f7b5a06b5b.png">

To test
* install web log sample data
* Add runtime geo point field to data view
* create and save map with runtime geo point field.
* remove runtime field from data view
* add map to dashboard. Verify map is displayed and error is surfaced in map legend
